templates/skeleton/app/routes/blogs.$blogHandle.$articleHandle.tsx

```diff
@@ -40,6 +40,7 @@ export default function Article() {
     <div className="article">
       <h1>
         {title}
+        <br />
         <span>
           {publishedDate} &middot; {author?.name}
         </span>
```

Comment:
Maybe better to use `<div>` instead of `<br>` + `<span>`? Or perhaps even extract the date and author outside of `<h1>`?